sql/moz-fx-data-shared-prod/telemetry_derived/experiment_search_aggregates_live_v1/view.sql

```diff
@@ -8,10 +9,36 @@ WITH all_searches AS (
     window_start,
     window_end,
     ad_clicks_count,
+    0 AS search_with_ads_count,
+    0 AS search_count,
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_fenix_derived.experiment_ad_clicks_count_live_v1`
+  WHERE
+    window_start > TIMESTAMP(DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY))
+  UNION ALL
+  SELECT
+    branch,
+    experiment,
+    window_start,
+    window_end,
+    0 AS ad_clicks_count,
     search_with_ads_count,
-    search_count
+    0 AS search_count,
```

Comment:
Turns out when combining all materialized views we are running into the following limitation: https://cloud.google.com/bigquery/docs/materialized-views-faq#join_and_union_all

When a base table scan appears multiple times in a query plan (like it's the case here, for each metric the same `metrics` base table gets scanned) the actual base tables get scanned and queried from, so querying `experiment_search_aggregates_live` becomes very expensive.